packages/lit/README.md

```diff
@@ -0,0 +1,16 @@
+# Lit 2.0
+
+Fast, lightweight web components
+
+## About this package
+
+The `lit` package contains everything needed to build Lit components with the LitElement base class, lit-html templates, and all first-party lit-html directives.
+
+Modules:
```

Comment:
You didn't mention:
* `directive-helpers.js`
* `disconnectable-directive.js`
* `hydrate-support.js`

Reply:
oh yeah, this reminds me: do we want to move `directive.ts`, etc., to live under `directives/`? That might be nicer when there's three top-level directive related files and the top level isn't lit-html specific anymore.

Reply:
@sorvell any thoughts?